Formula/heartbeat.rb

```diff
@@ -82,7 +82,13 @@ def post_install
     end
     sleep 5
     assert_match "hello", pipe_output("nc -l #{port}", "goodbye\n", 0)
+
     sleep 5
-    assert_match "\"status\":\"up\"", (testpath/"heartbeat/heartbeat").read
+    assert_match "first_start", (testpath/"data/meta.json").read
+
+    # test against `heartbeat-<calendar>.ndjson`
+    Dir.glob(testpath/"data/heartbeat-*.ndjson").each do |file|
```

Comment:
```suggestion
    testpath.glob("data/heartbeat-*.ndjson").each do |file|
```
Also, let's use `JSON.parse` to read JSON instead of parsing it as text.